documentation/docs/mock-apps/services/process-digital-identity-anchor.md

```diff
@@ -0,0 +1,86 @@
+---
+sidebar_position: 27
```

Comment:
I think this `sidebar_position: 27` might overlap with the `Merge To Local Storage` document's sidebar_position. Could you double check it?”

Reply:
Good catch!